usaspending_api/disaster/v2/views/object_class/object_class_result.py

```diff
@@ -0,0 +1,67 @@
+from dataclasses import dataclass, field
+from dataclasses_json import dataclass_json
+from typing import Dict
+
+from usaspending_api.common.data_classes import Pagination
+from usaspending_api.disaster.v2.views.data_classes import Collation, Element
+
+
+class MinorClass(Element):
```

Comment:
While MinorClass logically follows under MajorClass, ObjectClass is the correct term